architectures

```diff
@@ -1,3 +1,4 @@
 bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
+s390x default,alpine,onbuild,slim,stretch
```

Comment:
Yeah, once https://github.com/docker-library/official-images/blob/master/library/alpine references `s390x`, we're good to add that back. :+1: